src/main/java/com/conveyal/datatools/manager/models/transform/ZipTransformation.java

```diff
@@ -0,0 +1,12 @@
+package com.conveyal.datatools.manager.models.transform;
+
+/**
+ * This is an abstract class that represents a transformation that should apply to a GTFS in zip form. In other
+ * words, subclasses will provide a transform override method that acts directly on the zip file. Sample fields
+ * csvData and sourceVersionId can be used to reference replacement string or file that should be passed to the target
+ * zip file of the transformation.
+ */
+public abstract class ZipTransformation extends FeedTransformation {
```

Comment:
Not sure how much value this file adds. Especially since the `csvData` field isn't even used by the `ReplaceFileFromVersionTransformation` transformer. Another option might be to have a `FeedTransformation` have an enumeration of the stage of when the transformation is supposed to take place.